examples/machining.yaml

```diff
@@ -0,0 +1,430 @@
+# Example: Machine shop, one step of threading a part
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - alice: https://alice.example/
+    bob: https://bob.example/
+    machine: https://machine-shop.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/machining.yaml
+'@graph':
+
+  # recipe
+
+  - '@id': machine:a7e94dde-2997-451e-a7e1-8d364d5e6dee
+    '@type': ResourceSpecification
+    name: Threaded part 87
+    unit: unit:Number
+    substitutable: True
+
+  - '@id': machine:2d350055-acba-4e17-92f8-8b126272ee40
+    '@type': ResourceSpecification
+    name: Unthreaded part 45
+    unit: unit:Number
+    substitutable: True
+
+  - '@id': machine:07819789-dd51-44c3-b35c-9210165bc832
+    '@type': ResourceSpecification
+    classifiedAs: https://www.wikidata.org/wiki/Q2211295 # drilling machine
+    name: Drill Press
+    unit: unit:Number
+    unitOfUse: unit:Hour # want a better way to do this!
+    substitutable: True
+
+  - '@id': machine:c404cff5-19c7-453d-b0d8-d8e96055bd0b
+    '@type': RecipeProcess
+    skos:note: Thread a part
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 2
+    scalingFormula: D*M
+
+  - '@id': machine:c7897c39-7f05-4a5d-a487-80e130a2414b
+    '@type': RecipeFlow
+    action: produce 
+    resourceConformsTo: machine:a7e94dde-2997-451e-a7e1-8d364d5e6dee # threaded part
+    quantifiedAs:
```

Comment:
ok, in some older example i experimented with something similar, i think i called it `atomicQuantity` following the original meaning of *atom*. this also seems to answer my question above about `unitOfUse` in a way that if one wants to put some *atomic* / *quantized* constrains some other construct would take responsibility of that.